olive/data/component/pre_process_data.py

```diff
@@ -197,3 +197,73 @@ def text_generation_huggingface_pre_process(
         return text_gen_corpus_pre_process(dataset, tokenizer, all_kwargs)
     else:
         return text_gen_pair_pre_process(dataset, tokenizer, all_kwargs)
+
+
+@Registry.register_pre_process()
+def audio_classification_pre_process(
+    dataset,
+    model_name: str,
+    input_cols: list,
```

Comment:
the list typing hint is only supported after Python 3.9. we need support 3.8 from the time being. I suggest to use typing for back compatibility.

Reply:
Thanks! Fixed.